docker/images/dev-cli/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# Creates an image for skycoin development
+FROM golang:1.10.0-stretch
+
+# Installs software
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        cmake \
+        libpcre3-dev \
+        gdbserver \
+        gdb \
+        vim \
+        ctags \
+        vim-scripts \
+        screen \
+        sudo \
+        doxygen \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* 
+
+# Installs Criterion
+RUN git clone --recurse-submodules -j8 https://github.com/skycoin/Criterion && \
+    cd /go/Criterion && \
+    cmake . && \
+    make install && \
+    rm -r /go/Criterion
+
+RUN wget http://prdownloads.sourceforge.net/swig/swig-3.0.12.tar.gz \
+    && tar xzf swig-3.0.12.tar.gz \
+    && cd swig-3.0.12 \
+    && sh configure \
+    && make && make install \
+    && cd ../; rm -r swig-3.0.12*
+
+# Installs go development tools
+RUN go get -u github.com/derekparker/delve/cmd/dlv  && \
+    go get -u github.com/FiloSottile/vendorcheck && \
+    go get -u github.com/alecthomas/gometalinter && \
+    gometalinter --vendored-linters --install && \
+    go get -u github.com/zmb3/gogetdoc && \
+    go get -u golang.org/x/tools/cmd/guru && \
+    go get -u github.com/davidrjenni/reftools/cmd/fillstruct && \
+    go get -u github.com/rogpeppe/godef && \
+    go get -u github.com/fatih/motion && \
+    go get -u github.com/nsf/gocode && \
+    go get -u github.com/jstemmer/gotags && \
+    go get -u github.com/josharian/impl && \
+    go get -u github.com/fatih/gomodifytags && \
+    go get -u github.com/dominikh/go-tools/cmd/keyify
+
+# Install vim-go development tools
+RUN git clone https://github.com/fatih/vim-go /usr/share/vim/vim80/pack/dev/start/vim-go && \
```

Comment:
Ensure that these `vim-go` dependencies are installed . I had to install'em by executing `vim-go`'s `:GoInstallBinaries` command.

```
vim-go: gogetdoc not found. Installing github.com/zmb3/gogetdoc to folder $GOPATH/bin/
vim-go: fillstruct not found. Installing github.com/davidrjenni/reftools/cmd/fillstruct to folder $GOPATH/bin/
vim-go: motion not found. Installing github.com/fatih/motion to folder $GOPATH/bin/
vim-go: errcheck not found. Installing github.com/kisielk/errcheck to folder $GOPATH/bin/
vim-go: gotags not found. Installing github.com/jstemmer/gotags to folder $GOPATH/bin/
vim-go: impl not found. Installing github.com/josharian/impl to folder $GOPATH/bin/
vim-go: gomodifytags not found. Installing github.com/fatih/gomodifytags to folder $GOPATH/bin/
vim-go: keyify not found. Installing github.com/dominikh/go-tools/cmd/keyify to folder $GOPATH/bin/
vim-go: asmfmt not found. Installing github.com/klauspost/asmfmt/cmd/asmfmt to folder $GOPATH/bin/
vim-go: gometalinter not found. Installing github.com/alecthomas/gometalinter to folder $GOPATH/bin/
```

Reply:
Of course this needs to be done from the command line rather than through a `vim` command .

Reply:
I double checked . The former output was in my laptop . This is what I got inside the container 

```
vim-go: gorename not found. Installing golang.org/x/tools/cmd/gorename to folder
 /go/bin/
vim-go: asmfmt not found. Installing github.com/klauspost/asmfmt/cmd/asmfmt to folder
 /go/bin/
```